OneCommunityGlobal/HGNRest
# Luis unit test inventory controller get all inv in project

# Description
Unit Test for getAllInvInProjectWBS, postInvInProjectWBS, getAllInvInProject

## Related PRS (if any):
N/A
…

## Main changes explained:
- Unit tests for getAllInvInProjectWBS, postInvInProjectWBS, getAllInvInProject
- Integration tests for map locations router
…

## How to test:
1. check into current branch
2. do `npm install` 
3. run `npm run test inventoryController`
4. Verify all tests pass

## Screenshots or videos of changes:
<img width="1195" alt="Screenshot 2024-07-22 at 9 44 06 PM" src="https://github.com/user-attachments/assets/b926438d-bad6-45d8-9c3a-f65faddd18d0">

## Note:
Please note I submitted a previous PR with a few test, but I re-did them to relearn and improved it as well.
Here is the previous PR 921
https://github.com/OneCommunityGlobal/HGNRest/pull/921
